scst-store/using_encryption_and_connection.md

```diff
@@ -50,22 +50,32 @@ Where:
 
 - `PATH` is the location where you want to save the CA certificate. This file is used later when you [configure the CLI](cli_configuration.md).
 
-###<a id='editing-etchost'></a>Editing `/etc/hosts`
+### <a id='editing-etchost'></a>DNS resolution
```

Comment:
should probably change the id if we're changing the title/name

Reply:
done